### ISSUE TITLE
Clean DB

### ISSUE THREAD

**Issue by jby** [zulip/docker-zulip#187]
Since I'm not that proficient in docker/docker-compose it would be nice to have instructions on how to reset the DB and start fresh with a clean DB, while testing.

**Comment by timabbott** [comment]
Agreed, we should have that.  You can do that by removing everything in `/opt/docker` after shutting down the containers.

**Comment by mabushey** [comment]
@timabbott  Who is installing Docker in `/opt/docker` ? It's `/var/lib/docker` on every distro I've ever seen.

**Comment by jby** [comment]
I went with:
```
for f in $(docker image ls | awk '{print $3}');do docker image rm $f;done
```
But realize now that the DB is persistent under /opt/docker, will try to clean that as well.

**Comment by timabbott** [comment]
I expect upstream documentation is probably better than it was 5 years ago, but I suppose it could still be useful to write this down explicitly somewhere.

**Comment by funkyfuture** [comment]
nowadays a `docker compose down --volumes` should do it.
